scripts/README.md

```diff
@@ -0,0 +1,27 @@
+# Bash Scripts for Fornax Deployment
+### Node-A: Cloud Core Node 
+### Node-B: Edge Node with Control Plane 
+### Node-C: Edge Worker Node
+### 1. Generate ssh key in Edge Node with Control Plane (node-b) and copy ssh key ID to Cloud Core Node (node-a) & Edge Worker node (node-c):
+       ssh-keygen
+       ssh-copy-id (node-a IP)
+       ssh-copy-id (node-c IP)
```

Comment:
this part looks a bit confusing to me. where is user supposed to run these commands?

Reply:
also does (node-a IP) means it's a parameter to ssh-copy-id or something else? on line 17 below this format is being used but seemingly for different reason.